cas-server-core-api-ticket/src/main/java/org/jasig/cas/ticket/TicketState.java

```diff
@@ -21,7 +23,7 @@
      *
      * @return the last time the ticket was used.
      */
-    long getLastTimeUsed();
+    ZonedDateTime getLastTimeUsed();
```

Comment:
All tests pass. I looked into this to make sure but i don't remember what I ran into. I'll dig up a link later today.